concent_api/utils/logging.py

```diff
@@ -1,11 +1,15 @@
-from logging                        import getLogger
+from enum import Enum
 
 from golem_messages.message import FileTransferToken
-from golem_messages.message.base    import Message
+from golem_messages.message.base import Message
 
-from utils.helpers                  import get_field_from_message
+from core.models import Subtask
+from utils.helpers import get_field_from_message
 
-logger = getLogger(__name__)
+
+class MessageFields(Enum):
+    task_id = 'task_id'
```

Comment:
Please make it `TASK_ID = 'task_id'` and `SUBTASK_ID = 'subtask_id'`

Reply:
done